### PR TITLE
kvserver: deflake TestLeaseholdersRejectClockUpdateWithJump

### LINKED ISSUE #119362: kv/kvserver: TestLeaseholdersRejectClockUpdateWithJump failed
kv/kvserver.TestLeaseholdersRejectClockUpdateWithJump [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/14063253?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/14063253?buildTab=artifacts#/) on release-23.2.1-rc @ [898cd6a363fd47bb92a03bac216f9bed0f64bc08](https://github.com/cockroachdb/cockroach/commits/898cd6a363fd47bb92a03bac216f9bed0f64bc08):

```
=== RUN   TestLeaseholdersRejectClockUpdateWithJump
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/33e1d369c27b9c01b2b6009c561815a3/logTestLeaseholdersRejectClockUpdateWithJump1790262604
    test_log_scope.go:81: use -show-logs to present logs inline
    client_replica_test.go:191: remote wall time is too far ahead (500.000013ms) to be trustworthy
    panic.go:523: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/33e1d369c27b9c01b2b6009c561815a3/logTestLeaseholdersRejectClockUpdateWithJump1790262604
--- FAIL: TestLeaseholdersRejectClockUpdateWithJump (0.89s)
```

Parameters:
 - <code>TAGS=bazel,gss</code>
 - <code>stress=true</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/kv
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLeaseholdersRejectClockUpdateWithJump.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

Jira issue: CRDB-36167

### PR DESCRIPTION
**Pull request by pav-kv**
Fixes #119362
Epic: none
Release note: none

### CODE REVIEW

File: pkg/kv/kvserver/client_replica_test.go [hunk 1]
```diff
@@ -179,6 +179,11 @@ func TestLeaseholdersRejectClockUpdateWithJump(t *testing.T) {
 
 	manual.Pause()
 	ts1 := s.Clock().Now()
```
Comment (pav-kv) [inline pkg/kv/kvserver/client_replica_test.go:181]: A better fix is probably:

```golang
ts1 := manual.Now()
```

**Review by nvb — APPROVED** [review]
:lgtm:

> Reviewed 1 of 1 files at r2, 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/119370#-:-Nr3MKuQ9W9kaSyQpIzV:bnfp4nl)* status: <span>:shipit:</span> complete! 1 of 0 LGTMs obtained (waiting on @erikgrinaker)

**Comment by pav-kv** [comment]
bors r=nvanbenschoten

**Comment by pav-kv** [comment]
Will leave 22.2 without a backport since it's not merging cleanly. This is a rare enough flake.

**Pull request merged**